docs/configuration.md

```diff
@@ -19,6 +19,12 @@ const chart = eventDrops({ d3 });
 
 If you use EventDrops without any module bundler, just include D3 script before EventDrops, and everything should work out of the box.
 
+## id
+
+_Default: '' (not added to svg)_
```

Comment:
I agree with keeping the config simple, but I don't think I got what I was trying to achieve across. 
On zoomEnd you are able to retrieve the `d3.event` like this:
```js
        zoom: {
            onZoomEnd: () => {
                let event = d3.event;
                console.log(event.sourceEvent.target.id);
            }
        },
```
So the id is simply a way to distinguish between different charts (say two eventsDrop charts). So if you want to connect the zoom, you do it here. However, you also want to avoid an infinite loop. You could check the attributes, but they still have to be unique.
What you could do though is this:
```js
d3.event.sourceEvent.path[2].id
```
Which will get the id of the div, but that assumes that the div has an id.

What do you think? Just remove the id altogether or?

Reply:
Decided to remove it. It is quite easy to add id manually: `document.querySelector('#div svg').setAttribute('id', 'svg-id')`
Might make a blog/tutorial on how to connect the zoom when I got time (probs after 1.4.0 release).